collector/extract_observations.py

```diff
@@ -4,18 +4,20 @@
 def extract_observations(signal, num_samples, samples_per_observation,
                          sample_rate=44100):
```

Comment:
IIRC we don't need `sample_rate` anymore, since each row in the `signal` is a single sample.